cluster/manifests/skipper/deployment.yaml

```diff
@@ -331,6 +342,12 @@ spec:
             name: skipper-default-filters
             optional: true
 {{ end }}
+{{ if eq .ConfigItems.skipper_lua_scripts_enabled "true" }}
+        - name: lua
+          configMap:
+            name: skipper-ingress
+            optional: true
```

Comment:
Do we need optional here?

Reply:
Most other have `optional: true` so I stayed with that option. I guess it's better to have no lua but another instance than no new instance in case we need to scale-out and something happened why ever the configmap could not be mounted into the container.